ohmyk8s.yaml

```diff
@@ -118,7 +118,7 @@
       src: /tmp/sops
       remote_src: yes
       dest: /usr/local/bin/
-      mode: u+x
+      mode: uo+x
```

Comment:
What about this to allow the user group too?
```suggestion
      mode: a+x
```

Reply:
@hammady should this be applied to other binaries' mode?

Reply:
Yes, this is what I originally meant.

Reply:
Done.